ste/xfer-deleteFile.go

```diff
@@ -99,7 +99,11 @@ func doDeleteFile(jptm IJobPartTransferMgr, p pipeline.Pipeline) {
 	}
 
 	// Delete the source file
-	_, err := srcFileUrl.Delete(jptm.Context())
+	helper := &azureFileSenderBase{}
+	err := helper.DoWithOverrideReadOnly(jptm.Context(),
+		func() (interface{}, error) { return srcFileUrl.Delete(jptm.Context()) },
+		srcFileUrl,
+		jptm.GetForceIfReadOnly())
 	if err != nil {
```

Comment:
Just thinking, can this ever get called multiple times on a single file?

Perhaps we should have a sync.Once for clearing the flags, if so.

Reply:
It can't get called multiple times on a single file.